### PR TITLE
[WFLY-7193] SSL logging

### PR DESCRIPTION
**Pull request by honza889**
https://issues.jboss.org/browse/WFLY-7193
Related (but does not depend) to https://github.com/wildfly-security/wildfly-elytron/pull/502

### CODE REVIEW

File: src/main/java/org/wildfly/extension/elytron/FilteringKeyStoreService.java [hunk 1]
```diff
@@ -57,6 +59,15 @@ public void start(StartContext startContext) throws StartException {
             KeyStore unmodifiable = keyStoreService.getValue();
             KeyStore modifiable = keyStoreService.getModifiableValue();
 
+            if (ROOT_LOGGER.isTraceEnabled()) {
```
Comment (dmlloyd) [inline src/main/java/org/wildfly/extension/elytron/FilteringKeyStoreService.java]: Use tracef (see comments on https://github.com/wildfly-security/wildfly-elytron/pull/502 for more info).

**Comment by honza889** [comment]
Updated by comments.

File: src/main/java/org/wildfly/extension/elytron/FilteringKeyStoreService.java [hunk 2]
```diff
@@ -57,6 +59,15 @@ public void start(StartContext startContext) throws StartException {
...
+                ROOT_LOGGER.tracef("starting:%n" +
+                        "    aliasFilter = %s%n" +
```
Comment (darranl) [inline src/main/java/org/wildfly/extension/elytron/FilteringKeyStoreService.java]: I think we still need to remove the String concatenation here and move this onto a single line - then we can loose the wrapping isTraceEnabled

**Comment by honza889** [comment]
Updated - single-line log messages, no useless isTraceEnabled()

**Comment by darranl** [comment]
Sorry another one needing a rebase.

**Comment by honza889** [comment]
Rebased again :)

**Comment by darranl** [comment]
Sorry looks like your last PR merged conflicts with this one again ;-)

**Comment by honza889** [comment]
Rebased.

**Pull request merged**